scripts/case_selection/perform_case_selection.R

```diff
@@ -296,12 +304,14 @@ create_selection_matrix <- function(eligible_cohort, n_prod, n_pressure, n_sdv,
   
   # randomly disperse additional SDV cases among non-pressure
   col_sdv <- rep("", n_eligible)
+  set.seed(site_seed)
   idx_sdv <- sample((n_pressure+1):n_prod, n_sdv)
```

Comment:
Why do you need to set.seed so many times?

Reply:
You need to set seed every time before running sample to make the order reproducible.